build.gradle

```diff
@@ -160,4 +160,5 @@ test {
     if (!project.hasProperty('includePerfTests')) {
         exclude '**/YelpReviewsTest.class'
     }
+    systemProperties System.properties
```

Comment:
not sure what this line does

Reply:
This transfers the -D params given to ./gradlew to sys.properties in this particular task (test). 
If removed, and you ran `./gradlew test -Dflag:my_flag  --tests "org.apache.platypus.server.YelpSuggestTest", System.properties("my_flag") will result in a NullPointerExcption

Reply:
ok, thanks for explaining